liberty-managed/src/test/java/io/openliberty/arquillian/managed/needsmanagementmbeans/WLPInjectServletContextTest.java

```diff
@@ -1,6 +1,5 @@
 /*
- * JBoss, Home of Professional Open Source
- * Copyright 2010-2012, Red Hat Middleware LLC, and individual contributors
+ * Copyright 2018, IBM Corporation
```

Comment:
this is missing the ", and individual contributors" if it continues below with the "by the @authors tag"